src/vmm/src/lib.rs

```diff
@@ -564,7 +564,7 @@ impl Vmm {
             DEFAULT_ALLOC_POLICY,
         )?;
         self.fdt_builder.with_rtc(range.start(), range.len());
-        let range = mmio_from_range(range);
+        let range = mmio_from_range(&range);
```

Comment:
We should have this change in a separate commit. If that's too much hassle, we can keep it in this commit but we need to be explicit in the commit message to why this change is needed.